### ISSUE TITLE
does not actually record requests

### ISSUE THREAD

**Issue by ldeveber** [warrenday/graphql-network-inspector#116]
**Describe the bug**

does not work :(

**To Reproduce**

1. open dev panel and go to graphql tab
2. open dev panel on graphql tab
3. See error

**Expected behavior**
working?

**Screenshots**
![Screenshot 2023-10-31 at 11 49 28 AM](https://github.com/warrenday/graphql-network-inspector/assets/4457752/1ff4cd1c-b60a-4c16-86a4-e7e543b2a1a0)

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Mac OS Sonoma 14.0
- Chrome Version [e.g. 22] 118.0.5993.117

**Additional context**

```
extensions.js:1 Extension server error: Operation failed: Permission denied
t @ extensions.js:1
onmessage @ extensions.js:1
main.4f8f0ad3.js:2 Uncaught (in promise) TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
    at J (main.4f8f0ad3.js:2:1951306)
    at main.4f8f0ad3.js:2:1998793
    at c (main.4f8f0ad3.js:2:860483)
    at Generator._invoke (main.4f8f0ad3.js:2:860271)
    at Generator.next (main.4f8f0ad3.js:2:860912)
    at ee (main.4f8f0ad3.js:2:1951939)
    at o (main.4f8f0ad3.js:2:1952143)
J @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
c @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
ee @ main.4f8f0ad3.js:2
o @ main.4f8f0ad3.js:2
Promise.then (async)
ee @ main.4f8f0ad3.js:2
o @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
Bl @ main.4f8f0ad3.js:2
t.unstable_runWithPriority @ main.4f8f0ad3.js:2
qa @ main.4f8f0ad3.js:2
Ml @ main.4f8f0ad3.js:2
bl @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
t.unstable_runWithPriority @ main.4f8f0ad3.js:2
qa @ main.4f8f0ad3.js:2
Ka @ main.4f8f0ad3.js:2
$a @ main.4f8f0ad3.js:2
vl @ main.4f8f0ad3.js:2
ou @ main.4f8f0ad3.js:2
t.render @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
(anonymous) @ main.4f8f0ad3.js:2
legacy.js:1 Uncaught Error: No location name for view with id chrome-extension://ndlbedplllcgconngcnfmkadhokfaalnGraphQLNetwork
    at Pi.locationNameForViewId (legacy.js:1:121085)
    at $i.onContextMenu (legacy.js:1:142729)
    at di.tabContextMenu (legacy.js:1:115886)
locationNameForViewId @ legacy.js:1
onContextMenu @ legacy.js:1
tabContextMenu @ legacy.js:1
```

**Comment by warrenday** [comment]
Thanks for the detailed log @ldeveber. Does this error appear for all websites, or for a specific site?

**Comment by ldeveber** [comment]
> Thanks for the detailed log @ldeveber. Does this error appear for all websites, or for a specific site?

Good question! I originally tested a local app haha (http://localhost:3000/). It does work on our staging environment! But since I most often use it in local its definitely annoying.

I did a quick search to see if default permissions changed in chrome, and I found this: https://developer.chrome.com/blog/one-time-permissions/

It would be nice if it could prompt with what permission was missing/needed? Everything is set to defaults, so I'm not entirely sure what permissions I might need
